spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -104,7 +106,7 @@ Configure the Data Flow server URI with the following command (use the IP addres
  | |_| | (_| | || (_| | |  _| | | (_) \ V  V /    / / / / / /
  |____/ \__,_|\__\__,_| |_|   |_|\___/ \_/\_/    /_/_/_/_/_/
 
-1.0.0.BUILD-SNAPSHOT
+{dataflow-project-version}
```

Comment:
@ericbottard pointed out that this is supposed to be the dataflow server core version only. Hence, this is correct as is.